src/Infrastructure/LeanCode.ViewRenderer.Razor/RazorViewRendererModule.cs

```diff
@@ -12,8 +13,8 @@ public RazorViewRendererModule(RazorViewRendererOptions opts)
         this.opts = opts;
     }
 
-    protected override void Load(ContainerBuilder builder)
+    public override void ConfigureServices(IServiceCollection services)
     {
-        builder.Register(_ => new RazorViewRenderer(opts)).As<IViewRenderer>().SingleInstance();
+        services.TryAddSingleton<IViewRenderer>(_ => new RazorViewRenderer(opts));
```

Comment:
Hm, probably a dumb question but if it's supposed to be a singleton, why not instanciate one here already?

Reply:
Just a dumb rewrite :see_no_evil: 
The only difference I can see is that if one's invoked the method twice the constructor will be called only once, but I doubt that's it's a heavy call, so we can switch to explicit constructor.

Reply:
Right, I'm wondering if this isn't some legacy baggage because in general our Razor code is… not in good shape.